### DISCUSSION TITLE
How to set up notification alerts on Evebox

### DISCUSSION THREAD

**Discussion by grodish** [jasonish/evebox#264] [General]
I am trying to configure the alerts in evebox
What I am trying to do is that when I have an alert on evebox send me a notification

Then I add the following line to the file evebox.yaml

`notification: /home/suricata/scripts/notification_evebox.sh`

I have created the script in question

notification_evebox.sh:
`#!/bin/bash
read alert
echo "$alert" >> archivo.txt
`
but when I receive the alert in evebox it does not send me any notification

Could someone tell me how to do this?

**Comment by jasonish** [comment]
There is currently no external notification mechanism in EveBox, so the above won't work.

How do you add events to EveBox?

**Reply by grodish** [reply]
I am currently using evebox to read the eve.json file 

`input:
  filename: "/var/log/suricata/eve.json"`

I generate the eve.json file with trafr(mikrotik) and suricata

**Reply by jasonish** [reply]
Ok, so for now there is no option to generate external alerts.  If using Logstash it might have its own features.

I think its a reasonable feature request, but might take a while to get to.
